app/mailers/application_mailer.rb

```diff
@@ -13,9 +13,14 @@ def self.should_send_internal_emails?
 
   private
 
-  def add_image(image_file_name)
-    path = Rails.root.join("app/assets/images/mailer/#{image_file_name}")
-    attachments.inline[image_file_name] = File.read(path)
+  def add_image(image_file_name, source = "mailer")
```

Comment:
i would actually prefer we remove the `source` param and be more explicit about `image_file_name`, maybe renaming that param to `image_path`.

then in this method body:
`    path = Rails.root.join("app/assets/images/#{image_path}")`

and change the previous usages of `add_image` such as `add_image("logo-reddit.png") to `add_image("mailer/logo-reddit.png")`

the goal is to minimize logic and help readers of the code.